en/Sorting Algorithms/Merge Sort.md

```diff
@@ -18,9 +18,17 @@ Average - O(n log n)
 Worst case - O(n log n)
 ```
 
+The time complexity can be obtained by solving the below recurrence relation:
+
+$$T(n) = 2T\left(\frac{n}{2}\right) + An$$
```

Comment:
Is `A` defined anywhere? Is it just an arbitrary constant? This is unclear currently.

Reply:
So should I define `A` like solve the whole equation? Would that be appreciated?

Reply:
I understand. Will update with a proper derivation also define all used constants